src/robusta/core/playbooks/internal/crds.py

```
@@ -0,0 +1,360 @@
+                rows=rows,
+                headers=headers
+            )
+
+            finding.add_enrichment([table_block])
+
+        event.add_finding(finding)
+
+    except Exception:
+        msg = f"Error running fetch_resource_events for {params}"
+        logging.exception(msg)
+        raise ActionException(ErrorCodes.ACTION_UNEXPECTED_ERROR, msg)
```

Ruff (0.14.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)